components/time/src/zone/zone_name_timestamp.rs

```diff
@@ -184,7 +184,7 @@ impl serde::Serialize for ZoneNameTimestamp {
         #[cfg(feature = "alloc")]
         if serializer.is_human_readable() {
             let date_time = self.to_zoned_date_time_iso();
-            let year = date_time.date.extended_year();
+            let year = date_time.date.monotonic_year();
```

Comment:
nit: for ISO, maybe use this instead?

```suggestion
            let year = date_time.date.era_year().year;
```